org.supertux.SuperTux-Milestone1.json

```diff
@@ -28,6 +28,13 @@
                     "type": "git",
                     "url": "https://github.com/SuperTux/supertux.git",
                     "tag": "v0.1.4"
+                },
+                {
+                    "type": "shell",
+                    "commands": [ 
+                        "libtoolize",
+                        "autoreconf -vfi" 
```

Comment:
it's on the wrong module.

Reply:
aka, you can remove it and just update the `shared-modules` as it is smpeg that fails to build. It's usually a good practice to do so when upgrading runtime.

Reply:
(it builds locally after doing all of this)

Reply:
Thank you!